pkgs/tools/inputmethods/ibus-engines/ibus-mozc/default.nix

```diff
@@ -1,5 +1,6 @@
-{ clangStdenv, fetchFromGitHub, fetchpatch, which, ninja, python, gyp, pkgconfig
-, protobuf, ibus, gtk2, zinnia, qt5, libxcb, tegaki-zinnia-japanese }:
+{ clangStdenv, fetchFromGitHub, which, ninja, python, gyp, pkgconfig
```

Comment:
There are now two `stdenv`s here, please reduce this to one.  I'd prefer to only have `stdenv` as an argument an then set `stdenv = clangStdenv` in `all-packages.nix`